docs/tools/event-listener.md

```diff
@@ -0,0 +1,29 @@
+# Event Listener
+
+ <iframe width="640" 
+         height="480" 
+         src="https://www.youtube.com/embed/gcT8Bp5oepo" 
+         title="Event Listener with Lit Protocol - Automate Web3 Signing" 
+         frameborder="0" 
+         allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+         allowfullscreen>
+ </iframe>
+
+
+The [Lit event listener](https://github.com/DIGITALAX/LitListenerSDK) allows you to create specific event-based triggers for executing Lit Actions.
+
+The SDK is designed around the principle of conditionally pre-approved chain reactions with account abstraction. This series of actions encrypted, conditional, often on-chain, can be chained together to trigger complex strategies.
+
+With the SDK, you'll be able to set three core condition functions: webhooks, on-chain events, and intervals.
+
+1. Webhooks
+The SDK is designed to query (request) and monitor information from specific APIs. In a blockchain context, this might include tracking price changes of a specific cryptocurrency or updates from a decentralized app.
```

Comment:
Adding some information on what a `hook` is, specifically that is user defined behvior executing when a predefined condition is met.